modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/thread/ThreadMdcUtil.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator.thread;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import org.slf4j.MDC;
+
+/**
+ * Bean to get mdc logging info.
+ */
+public final class ThreadMdcUtil {
+
+    private static final String TRACE_ID = "TRACE_ID";
+
+    /**
+     * @return uuid
+     */
+    public static String generateTraceId() {
+        return UUID.randomUUID().toString();
```

Comment:
we are creating a UUID for each register request while creating the DB entry. We can reuse the same to be consistent here.